### PR TITLE
Get playlists endpoint

### LINKED ISSUE #11: Playlist Endpoints - View all playlists
As a user,
When I request GET /api/v1/playlists,
It returns all the playlists in the database ALONG with their associated songs.

Response in this format:

[
    {
        "id": 1,
        "playlist_name": "Favorite songs of all time",
        "songs": [
          {
            "id": 1,
            "name": "We Will Rock You",
            "artist_name": "Queen"
            "genre": "Rock",
            "song_rating": 88
          },
          {
            "id": 2,
            "name": "Careless Whisper",
            "artist_name": "George Michael"
            "genre": "Pop",
            "song_rating": 93
          }
        ]
    },
    {
        "id": 2,
        "name": "Other amazing songs",
        "songs": [
          {
            "id": 1,
            "name": "We Will Rock You",
            "artist_name": "Queen"
            "genre": "Rock",
            "song_rating": 88
          },
          {
            "id": 2,
            "name": "Careless Whisper",
            "artist_name": "George Michael"
            "genre": "Pop",
            "song_rating": 93
          },
        ]
    },
]

### PR DESCRIPTION
**Pull request by JLing88**
____ Wrote Tests
__x__ Implemented
__x__ Reviewed
# Neccesary checkmarks:
- [x] All Tests are Passing

- [] The code will run locally

## Type of change
- [x] New feature
- [] Bug Fix

# Implements/Fixes:
* Add route for api/v1/playlists
* Add SQL statement to retrieve playlists and their associated songs
* Format JSON response according to spec
closes #11 

# Check the correct boxes

- [x] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

# Please Include a link to a gif of your feelings about this branch
Link:

### CODE REVIEW

**Comment by JLing88** [comment]
@amesee, can you please check out the SQL query for GET api/v1/playlists. Does that look/feel ok you? Thanks!

**Comment by ameseee** [comment]
Hi @JLing88 I like that you are getting this done in one query, looks good.

**Comment by JLing88** [comment]
@ameseee Thank you!

File: src/app.js [hunk 1]
```diff
@@ -89,4 +89,25 @@ app.delete('/api/v1/songs/:id', (request, response) => {
   });
 });
 
-module.exports = app;
\ No newline at end of file
+app.get('/api/v1/playlists', (request, response) => {
+  database.raw
+  (`
+    SELECT playlists.id, playlists.name,
```
Comment (dmeskis) [inline src/app.js]: SQL MASTER OVER HERE!

**Pull request merged**